### PR TITLE
add README and installation instructions

### PR DESCRIPTION
**Pull request by woss**
I wanted to read this book but there were no instructions on how to install the deps and built it locally. After I managed to do that I decided to make a PR so others can benefit. 

Signed-off-by: Daniel Maricic <daniel@woss.io>

### CODE REVIEW

File: docs/book/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# cachepot distributed compilation quickstart
```
Comment (drahnr) [inline docs/book/README.md:1]: The title of this file should be about running a local instance of the book, not its content.

File: docs/book/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# Building
+
```
Comment (drahnr) [inline docs/book/README.md:4]:
```suggestion
```

File: docs/book/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+cargo install mdbook mdbook-linkcheck mdbook-mermaid
+```
+
+After they are successfully installed you can serve the book using
+
+```sh
+mdbook serve
+```
+
+which will be available on [http://127.0.0.1:3000](http://127.0.0.1:3000).
```
Comment (drahnr) [inline docs/book/README.md:17]:
```suggestion
which will be available on <http://127.0.0.1:3000>.
```

**Review by drahnr — CHANGES_REQUESTED** [review]
Thank you for your contribution! A few minor nits to be addressed, otherwise looks good to me!

File: docs/book/README.md (hunk 1 above)
Comment (woss) [reply docs/book/README.md]: title changed to `# How to run cachepot book?` which makes more sense.

File: docs/book/README.md [hunk 4]
```diff
@@ -1,8 +1,6 @@
-# cachepot distributed compilation quickstart
+# How to run cachepot book?
```
Comment (drahnr) [inline docs/book/README.md:1]:
```suggestion
# The Cachepot Book
```

**Review by drahnr — APPROVED** [review]

File: docs/book/README.md (hunk 4 above)
Comment (woss) [reply docs/book/README.md]: do you want me to change it?

**Pull request merged**